packages/lit-dev-content/src/components/litdev-tutorial.ts

```diff
@@ -334,17 +429,53 @@ export class LitDevTutorial extends LitElement {
   }
 
   protected renderFooter() {
-    return html`<div id="tutorialFooter">
-      <litdev-icon-button @click=${this._onClickSolve}>
-        ${solveIcon} Solve
-      </litdev-icon-button>
+    const snackbarLabel = `The code has ${
+      this._checkStatus === 'PASSED' ? 'passed' : 'failed'
+    } the checks${
+      this._validationMessage ? ` – ${this._validationMessage}` : '.'
+    }`;
 
-      <litdev-icon-button @click=${this._onClickReset}>
-        ${resetIcon} Reset
-      </litdev-icon-button>
+    return html`<div id="tutorialFooter">
+        <litdev-icon-button @click=${this._onClickSolve}>
+          ${solveIcon} Solve
+        </litdev-icon-button>
+
+        ${this._manifestTask.render({
+          complete: (manifest) =>
+            !manifest.steps[this._idx].checkable
+              ? nothing
+              : html` <litdev-icon-button
```

Comment:
Maybe factor this out into a `_renderCheckButton`? Indentation is a little severe.